mscore/importmxmlpass2.h

```diff
@@ -147,10 +161,11 @@ class MxmlLogger;
 
 using SlurStack = std::array<SlurDesc, MAX_NUMBER_LEVEL>;
 using TrillStack = std::array<Trill*, MAX_NUMBER_LEVEL>;
-using BracketsStack = std::array<SLine*, MAX_BRACKETS>;
-using DashesStack = std::array<SLine*, MAX_DASHES>;
```

Comment:
As `MAX_DASHES` seems to be not used anymore, could you please remove it also from `musicxml.h`?
https://github.com/musescore/MuseScore/blob/8f0c2dd908c7cf75d28153f69f15885a15b82447/mscore/musicxml.h#L52

Reply:
Will do, but am unable to do it right now (real life obligations).

Reply:
Done